CMakeLists.txt

```diff
@@ -139,7 +139,7 @@ option(USE_HB "Use HammerBlade" ON)
 option(USE_HB_EMUL "Use HammerBlade Emulation Layer" OFF)
 option(PROFILE_ATEN "Turn on ATen kernel profiling" ON)
 cmake_dependent_option(
-  HB_REDISPATCH "Enable conditional redispatch to HB" ON
+  HB_REDISPATCH "Enable conditional redispatch to HB" OFF
```

Comment:
looks like this change is causing some of the failures we have seen on Travis

Reply:
Good point ! Thanks !